src/lib/intent/editor-layout/editor-layout-change-field-group-id.ts

```diff
@@ -0,0 +1,55 @@
+import Intent from '../base-intent'
+import { PlanMessage } from '../../interfaces/plan-message'
+import chalk from 'chalk'
+import { EditorLayoutChangeFieldGroupIdAction } from '../../action/editor-layout/editor-layout-update-field-group'
+
+export default class EditorLayoutChangeFieldGroupIdIntent extends Intent {
+  isEditorInterfaceIntent() {
+    return true
+  }
+  isEditorLayoutUpdate() {
+    return true
+  }
+  isGroupable() {
+    return true
+  }
+  groupsWith(other: Intent): boolean {
+    return other.isGroupable()
+      && other.isEditorLayoutUpdate()
+      && this.isSameContentType(other)
+  }
+  endsGroup(): boolean {
+    return false
+  }
+  shouldSave(): boolean {
+    return true
+  }
+  shouldPublish(): boolean {
+    return false
```

Comment:
Are updates to editor interfaces immediate?

Reply:
What do you intend as immediate? As far as I understood, all intents changes within a group are applied together.

Reply:
Immediate = they don't need to be published. I'm not sure about the lifecycle of editor_interfaces so just want to clarify.

Reply:
Yep, no publication of editor interfaces